3.distributed-calculator/README.md

```diff
@@ -9,7 +9,85 @@ This sample shows method invocation and state persistent capabilities of Dapr th
 
 The front-end application consists of a server and a client written in [React](https://reactjs.org/). 
 Kudos to [ahfarmer](https://github.com/ahfarmer) whose [React calculator](https://github.com/ahfarmer/calculator) 
-sample was used for the client. The following architecture diagram illustrates the components that make up this sample: 
+sample was used for the client.
+
+## Running locally
+
+We will be starting the four operator apps (add, subtract, multiply and divide) along with the dapr sidecar locally and then run the front end app which persists the state in a local redis state store.
+
+1. Add App - Open a terminal window and navigate to the go directory and follow the steps below:
+   - Install the gorilla/mux package: Run:
+      ```
+      go get -u github.com/gorilla/mux
+      ```
+    - Build the app. Run:
+      ```
+      go build app.go
+      ```
+    - Run dapr using the command:
+      ```
+      dapr run --app-id addapp --app-port 6000 --port 3503 ./app
+      ```
+2. Subtract App - Open a terminal window and navigate to the csharp directory and follow the steps below:
+    - Set environment variable to use non-default app port 7000
+      ```
+      Linux/Mac OS:
+      export ASPNETCORE_URLS=7000
```

Comment:
Should this be `http://127.0.0.1:7000`?

Reply:
No...just 7000

Reply:
When I tried just the `7000` I am getting the following error:

```bash
== APP == crit: Microsoft.AspNetCore.Server.Kestrel[0]

== APP ==       Unable to start Kestrel.

== APP == System.FormatException: Invalid url: '7000'
...
```
but no such problem with `http://127.0.0.1:7000`.

Reply:
Thanks! Checked my settings and you were right